apps/mobile/src/locales/index.ts

```diff
@@ -25,3 +27,11 @@ export const LOCALES = [
 export const LOCALE_CODES = LOCALES.map(({ locale }) => locale);
 
 export const DEFAULT_LOCALE = 'en';
+
+export function initiateI18n() {
```

Comment:
Can you please educate me on this? Is it to better load more locales?

Reply:
This was just copied from another folder, we had i18n/ and locales/ before. Figure one is enough to tidy root